ui/tests/acceptance/clients/counts/sync-test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import clientsHandler from 'vault/mirage/handlers/clients';
+import { visit, click, currentURL } from '@ember/test-helpers';
+import authPage from 'vault/tests/pages/auth';
+import { SELECTORS } from 'vault/tests/helpers/clients';
+
+module('Acceptance | clients | sync', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    clientsHandler(this.server);
```

Comment:
TODO there seems to be quite a bit more setup i need to do here to ensure charts are actually rendered on the page when the feature is activated. i've been using [this test](https://github.com/hashicorp/vault/blob/164a1af54a3a732aa2252de6fe7c02567622cf8a/ui/tests/acceptance/clients/counts-test.js#L1-L60) as a reference but haven't quite figured out which things are relevant to set up or not.